app/components/SearchHistory/index.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import SearchHistoryItem from 'components/SearchHistoryItem';
+import { SearchHistoryList, SearchHistoryLabel, SearchHistoryWrapper } from './styles';
+
+const SearchHistory = ({ searchHistory }) =>
+  searchHistory.length > 0 && (
```

Comment:
Shouldn't this check be performed on a higher level. Say, the component that renders the `SearchHistory` component?

Reply:
Moving up this logic means it would have to live in the SearchHistory container. That's not a good separation of concerns. Containers should only be used for data flow and routing actions to presentational components.